02_configure_host.sh

```diff
@@ -273,36 +270,50 @@ elif [[ "$reg_state" != "running" ]]; then
 fi
 sleep 5
 
+detect_mismatch(){
+  local LOCAL_IMAGE="$1"
+  local REPO_PATH="$2"
+  if [[ -z "${LOCAL_IMAGE}" ]] || [[ "${LOCAL_IMAGE}" == "${REPO_PATH}" ]]; then
+    echo "Local image: ${LOCAL_IMAGE} and repo path: ${REPO_PATH} are matching!"
```

Comment:
Is this printout really necessary. We already know if they don't match since it terminates the whole thing.

Reply:
I would like everything as verbose as possible, it is very hard to find exact stages in the dev-env logs, this would help during debugging, we would at least know that we have reached and executed the miss match detection.